samples/13.core-bot/src/main/java/com/microsoft/bot/sample/core/DialogAndWelcomeBot.java

```diff
@@ -0,0 +1,66 @@
+package com.microsoft.bot.sample.core;
+
+import com.codepoetics.protonpack.collectors.CompletableFutures;
+import com.microsoft.applicationinsights.core.dependencies.apachecommons.io.IOUtils;
+import com.microsoft.applicationinsights.core.dependencies.apachecommons.lang3.StringUtils;
+import com.microsoft.bot.builder.*;
+import com.microsoft.bot.dialogs.Dialog;
+import com.microsoft.bot.schema.Activity;
+import com.microsoft.bot.schema.Attachment;
+import com.microsoft.bot.schema.ChannelAccount;
+import com.microsoft.bot.schema.Serialization;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+public class DialogAndWelcomeBot extends DialogBot {
+    /**
+     * Creates a DialogBot.
+     *
+     * @param withConversationState ConversationState to use in the bot
+     * @param withUserState         UserState to use
+     * @param withDialog            Param inheriting from Dialog class
+     */
+    public DialogAndWelcomeBot(ConversationState withConversationState, UserState withUserState, Dialog withDialog) {
+        super(withConversationState, withUserState, withDialog);
+    }
+
+    @Override
+    protected CompletableFuture<Void> onMembersAdded(
+        List<ChannelAccount> membersAdded, TurnContext turnContext
+    ) {
+        return turnContext.getActivity().getMembersAdded().stream()
+            .filter(member -> !StringUtils
+                .equals(member.getId(), turnContext.getActivity().getRecipient().getId()))
+            .map(channel -> {
+                Attachment welcomeCard = createAdaptiveCardAttachment();
+                Activity response = MessageFactory.attachment(welcomeCard);
```

Comment:
`MessageFactory.attachment` does not receive `ssml`?

See [C#](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/13.core-bot/Bots/DialogAndWelcomeBot.cs#L33)

Reply:
We need to use [this](https://github.com/microsoft/botbuilder-java/blob/ca5d5575259d140c9b33297aef970d7fa2d46906/libraries/bot-builder/src/main/java/com/microsoft/bot/builder/MessageFactory.java#L238)